src/map/utils/battleutils.cpp

```diff
@@ -898,6 +898,13 @@ namespace battleutils
                     {
                         PAttacker->takeDamage(spikesDamage, PDefender, ATTACK_TYPE::MAGICAL, DAMAGE_TYPE::LIGHT);
                     }
+                    else if (PDefender->objtype == TYPE_TRUST)
```

Comment:
This block shouldn't be necessary anymore due to the change in `TakePhysicalDamage` go ahead and try without it

Reply:
Confirmed. Staging reversion.

![image](https://user-images.githubusercontent.com/99501957/198134190-17b0122d-3746-4d26-bb2e-cd7f2183a882.jpeg)